hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/ITestS3AEndpointRegion.java

```diff
@@ -510,6 +514,16 @@ public void testCentralEndpointAndNullRegionFipsWithCRUD() throws Throwable {
     assertOpsUsingNewFs();
   }
 
+  /**
+   * Skip the test if the region is sa-east-1.
+   */
+  private void skipCrossRegionTest() throws IOException {
+    String region = getFileSystem().getS3AInternals().getBucketMetadata().bucketRegion();
+    if (SA_EAST_1.equals(region)) {
```

Comment:
needs a story for testing with third party stores. There I have a region like "unknown" or "test" (but not empty string)

Maybe, rather than be clever here, declare that for testing the region should be declared as "non-aws", with
* the new string stuck in S3ATestConstants
* the skip test here expanded to check for it
* the testing.md doc updated to cover this

everything else will take any string as a region, at least of those I've tested

Reply:
It makes sense to me.
Ack.